### DISCUSSION TITLE
[tauri v2] Questions about `csp`

### DISCUSSION THREAD

**Discussion by percy507** [tauri-apps/tauri#7881] [Q&A] [answered]
- What is the default `csp` of tauri?
- When set `csp: null`, does this means tauri will disable csp?

In my app, its csp is `null`. And I can open it on macOS correctly, but when open it on Windows, I got white screen and the console reports below errors. I'm curious why.

```bash
POST https://ipc.localhost/plugin%3Aevent%7Clisten net::ERR_CONNECTION_REFUSED
POST https://ipc.localhost/plugin%3Awindow%7Cis_fullscreen net::ERR_CONNECTION_REFUSED
```

**Comment by FabianLars** [comment, marked as answer]
`csp: null` indeed disables the csp. I _think_ that error means some tauri package may be outdated since the protocol on windows switched to `http` in v2 recently.

**Reply by percy507** [reply]
Thanks. The error has been fixed on `2.0.0-alpha.14`.
